tests/test_claim_filing_accuracy_all_cases.py

```
@@ -0,0 +1,338 @@
+        GK_SPECIAL_URL,
+    )
+    draft = _draft()
+
+    finalize_professional_claim(_case_context(), _research(wrong_part), draft)
+
+    assert not draft.legal_basis
+    assert draft.status == VerificationStatus.NEEDS_VERIFICATION
+    assert any(note.startswith(LEGAL_GROUNDING_PREFIX) for note in draft.verification_notes)
+
+
+def test_ambiguous_same_article_parts_are_not_auto_corrected(grounding_corpus: Path) -> None:
```

Ruff (0.16.2)

[warning] Unused function argument: `grounding_corpus`

(ARG001)